src/libopal.c

```diff
@@ -2312,6 +2312,13 @@ gen_asm_code (node_s *ast)
       gen_asm_code(ast->left);
       add_asm_code(ast->node_type, 0, NULL);
       break;
+    case nd_Ident:
+    case nd_Integer:
+    case nd_String:
+    case nd_Assign:
```

Comment:
1. Generate asm code for left child.
2. Get `location_offset` of left child's integer by calling `add_var` and add asm code `asm_Store`